pytest_testrail/__init__.py

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env python3
```

Comment:
Why the `shebang`? This code is not meant to be directly executed, right?

Reply:
I systematically follows the [documentation recommendation](https://docs.python.org/3/using/unix.html?highlight=shebang#miscellaneous) but, you're right, this code will not be directly executed. Moreover, this code is not Python3 specific.